### PR TITLE
Training statement

### PR DESCRIPTION
**Pull request by bobturneruk**
Statement on free at point of use training. Hopefully will encourage attendance / cancellation rather than no-shows.

### CODE REVIEW

File: pages/training/statement/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+permalink: /training/statement
+---
+
+We are delighted to be able to make free at point of use training available to the research community, to enable better software and more open, reproducible research. However, **free at point of use training is not free**. The cost of a course can easily run to thousands of pounds, if preparation costs are taken into account.
+
+**If you sign up for a course, please make sure you either attend or cancel your booking.** Bookings can usually be cancelled using [eventbrite.com](https://www.eventbrite.com) or, failing that, by emailing [rse@sheffield.ac.uk](mailto:rse@sheffield.ac.uk).
+
+Running courses that are not fully attended wastes our funding (which is provided by taxpayers, charities and students, amongst others) and reduces our collective capacity to improve research outputs and researcher experiences.
+
+Failure to attend a booked course may result in you being reported to your department.
```
Comment (annakrystalli) [inline pages/training/statement/index.md]: I'm not sure about this, nor do I feel departments will be interested in policing our courses. Rather, I was thinking we could state that _"persistent failure to attend booked courses might result in being excluded from future training opportunities."_

Comment (bobturneruk) [reply pages/training/statement/index.md]: I like that. It's suitably non-specific. 

I do think it's appropriate for departmental administrators to share the burden of non-attendance, though. Perhaps something to reconsider, later.

Comment (bobturneruk) [reply pages/training/statement/index.md]: Fixed with #2d9e450578c55682279a85caa117cc376118db62

Comment (annakrystalli) [reply pages/training/statement/index.md]: Nice one

**Review by annakrystalli — APPROVED** [review]

**Pull request merged**